app/wama.web/Controllers/UserToolController.cs

```diff
@@ -236,7 +236,8 @@ public IActionResult AddCertification(string memberId)
         public async Task<IActionResult> ViewCertifications(string memberId)
         {
             SetActiveConsoleTool(Constants.ADMIN_CONSOLE_USERS_PATRONS);
-            var Certifications = await _CertificationService.GetCertificationsAsync(memberId);
+            var Certifications = await _UserAccountService.GetUserAccountAsync(memberId);
+            ViewData["Certifications"] = await _CertificationService.GetCertificationsAsync(memberId);
```

Comment:
I am pretty sure, i put var certification to grab the certification data of some user, why did you change to grab User account into certification??
use of view data should be only used for passing info into view (html)

put it back please

if you requires of grabbing user account service, use a new var instead of changing certifications